cli/tests/test_journal.rs

```diff
@@ -0,0 +1,40 @@
+/// rexpect does not work on Windows.
+/// https://github.com/rust-cli/rexpect/issues/11
+#[cfg(not(target_os = "windows"))]
+mod tests {
+    use assert_cmd::cargo::cargo_bin;
+    use rexpect::error::*;
+    use rexpect::session::spawn_command;
+    use std::process;
+
+    #[test]
+    fn test_pragma_journal_mode_wal() -> Result<(), Error> {
+        let mut child = spawn_command(run_cli(), Some(1000))?;
+        child.exp_regex("limbo>")?; // skip everything until limbo cursor appear
+        child.exp_regex(".?")?;
+        child.send_line("pragma journal_mode;")?;
+        child.exp_string("wal")?;
+        child.send_line(".quit")?;
+        child.exp_eof()?;
+        Ok(())
+    }
+
+    #[ignore = "wal checkpoint not yet implemented"]
```

Comment:
WAL checkpointing is implemented -- what is preventing from enabling this test?

Reply:
I thought the Pager api was not available yet. actually it was.